### ISSUE TITLE
why "wrong transports" not always be 0?

### ISSUE THREAD

**Issue by charleshm** [data61/stellar-random-walk#5]
By using the routing table, the vertices would be sent to the right partition, but why "wrong transports" not always be 0?

**Comment by shps** [comment]
True, the "wrong transports" should be zero and if the "wrong transports" is greater than zero it implicates that a random walker had no progress after it was moved to another partition. It may imply that the routing table is not working properly. Can you please give more detail if you have been facing such a problem?
